cobra/test/test_io/test_yaml.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+"""Test functionalities provided by yaml.py"""
+
+from __future__ import absolute_import
+
+from os.path import join
+
+from cobra import io
+from cobra.test.test_io.conftest import compare_models
+
+
+def test_load_yaml_model(data_directory, mini_model):
```

Comment:
Loading the YAML at a lower level gives you a dictionary structure. It'd be great to test that object structure against the JSONSchema as well.